### PR TITLE
Preview column list

### PR DESCRIPTION
**Pull request by bambukasvi**

### CODE REVIEW

File: src/components/PreviewColumn.jsx [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+            </li>
+          );
+        }
+      )}
+    </ul>
+  );
+
+  return (
+    <div className={previewColumn}>
+      {header && <h2>{header.toUpperCase()}</h2>}
```
Comment (elisa-n) [inline src/components/PreviewColumn.jsx:66]: css property `text-transform: uppercase` is better for both code readability and screen readers

File: src/components/PreviewColumn.jsx [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+      {renderColumn()}
+      {header && (
+        <div className={linkArrow}>
+          <LinkWithArrow to="docs/test-article" type="secondary">
+            {`More ${header.toLowerCase()}`}
```
Comment (elisa-n) [inline src/components/PreviewColumn.jsx:71]: same here, prefer text-transform

File: src/styles/globals/_breakpoints.scss [hunk 3]
```diff
@@ -6,5 +6,5 @@ $breakpoints: (
   // tablets
   'lg': 1280px,
   // desktops
-  'xl': 1920px // xl desktops, tvs,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,
+  'xl': 1920px // xl desktops, tvs,,,,,,,
```
Comment (elisa-n) [inline src/styles/globals/_breakpoints.scss:9]: i think prettier is maybe adding these? 😂

Comment (bambukasvi) [reply src/styles/globals/_breakpoints.scss:9]: Yeah, too weird.. i just removed the whole comment :D

**Review by elisa-n — APPROVED** [review]

**Pull request merged**